examples/rllib/stabilizing_the_ring.py

```diff
@@ -86,12 +86,19 @@
     # parameters specifying the positioning of vehicles upon initialization/
     # reset (see flow.core.params.InitialConfig)
     initial=InitialConfig(),
+
```

Comment:
Hmmm, I'm not sure we want to add this as a flow_param; this adds a param that's rllib specific. Could we keep this same idea, but move it out of flow_params?